src/room/InCallView.tsx

```diff
@@ -98,7 +98,7 @@ export function ActiveCall(props: ActiveCallProps) {
   const livekitRoom = useLiveKit(props.userChoices, {
     sfuUrl: Config.get().livekit!.server_url,
     jwtUrl: `${Config.get().livekit!.jwt_service_url}/token`,
-    roomName: props.matrixInfo.roomName,
+    roomName: `{props.matrixInfo.roomName}({props.matrixInfo.roomId})`,
```

Comment:
I think the problem that gave rise to https://github.com/vector-im/element-call/issues/1165 isn't that the LiveKit room name wasn't unique - it was that it wasn't stable. Including the Matrix room name in this string means that the result still won't be stable (it could either change over time, or vary between different participants when in a room without an explicit name), so I would instead suggest a combination of room ID + call ID.

Reply:
Yeah, this make sense

Reply:
Oh, you're right, thanks! Fixed. Just to make sure that I did not mess up things: `groupCall.groupCallId` is the same as the `conf_id` (conference ID), right? (not the `callID` that was different for each participant who participated in a conference - the problem that we observed when testing `waterfall` several months ago)

Reply:
Yep, that's right